_src/Anti-Debug/techniques/misc.md

```diff
@@ -485,12 +486,20 @@ bool Generic::CheckWrittenPages2() const {
 {% endhighlight %}
 
 <hr class="space">
+<br />
+
+<h3><a class="a-dummy" name="ifeo-removal">8. IFEO removal</a></h3>
```

Comment:
Please add the code example along with corresponding entries you are describing, it can be the list or the table